uavcan/tunnel/2011.SerialConfig.uavcan

```diff
@@ -0,0 +1,7 @@
+#
+# This message carries baudrate for tunnel channel_id
```

Comment:
worth adding options to allow for inversion, flow control etc. we have room in an 8 byte packet, so 16 bit options would be worth having for future expansion